File: 2016/finale/filiali/att/filiali.c
#include <stdio.h>
#include <assert.h>

#define MAXN 1000000

int apri(int N, int F, int K[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int K[MAXN];

int main() {
    FILE *fr, *fw;
    int N, F, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(2 == fscanf(fr, "%d %d", &N, &F));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &K[i]));

    fprintf(fw, "%d\n", apri(N, F, K));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed2/finale/filiali/att/filiali.pas
function apri(N, F: longint; var K: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    apri := 42;
end;


const
    MAXN = 1000000;
var
    N, F, i : longint;
    K       : array[0..MAXN-1] of longint;
    fr, fw  : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N, F);
    for i:=0 to N-1 do
        read(fr, K[i]);

    writeln(fw, apri(N, F, K));
    close(fr);
    close(fw);
end.
